src/test/resources/expected_marc.jsonl

```diff
@@ -304,6 +304,14 @@
       "ind1" : " ",
       "ind2" : " "
     }
+  }, {
+    "257" : {
+      "subfields" : [ {
+        "a" : "France"
+      } ],
+      "ind1" : " ",
+      "ind2" : " "
+    }
```

Comment:
minor
I like the big integration test, it allows you to see the big picture
However I also suggest doing narrowly focused cases for converting individual fields
an example of such tests already exists for field 245, adding even this. Even the simplest test allows you to capture the logic and further narrow the search for a problem and add various scenarios in the future.